packages/amplify_datastore/example/pubspec.yaml

```diff
@@ -23,6 +23,10 @@ dependencies:
   #   path: ../../amplify_api
   amplify_flutter:
     path: ../../amplify_flutter
+  flutter_driver:
```

Comment:
suggestion: I would define the min flutter version as 2.2.0 in env block above like https://github.com/aws-amplify/amplify-flutter/blob/main/packages/amplify_api/example/pubspec.yaml#L10 . The integration test driver is only in 2.0.0 + and the versions between 2 and 2.2 have some null safety issue that sometimes involves adding an override dep, so I think stating for flutter 2.2.0 + technically the correct thing to do. In theory, could make CI integration easier later by avoiding cryptic error messages.

Reply:
Updated 👍